src/vfxt/src/createUiDefinition.json

```diff
@@ -284,7 +284,7 @@
               "type": "Microsoft.Common.DropDown",
               "label": "Use blob storage",
               "defaultValue": "true",
-              "toolTip": "If true, create the storage account and use that for the backing storage account.  If false, the minimal Avere vFXT cluster will be created.",
+              "toolTip": "If true, create a storage account and configure a new blob container as the cluster's backend storage. If false, you must attach storage after creating the cluster.",
```

Comment:
Correct, but it sounds like, "If true, then create a storage account and then go and configure a container" when it really says, "This option will create a storage account and a new container."

Reply:
updated